src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataGuidanceSection.tsx

```diff
@@ -61,11 +62,33 @@ const ReleaseDataGuidanceSection = ({ releaseId, canUpdateRelease }: Props) => {
 
   const handleSubmit = useFormSubmit<DataGuidanceFormValues>(
     async (values, helpers) => {
+      const sanitizedValues = {
+        ...values,
+        subjects: values.subjects.map(value => {
+          return {
+            ...value,
+            content: sanitizeHtml(value.content, {
```

Comment:
Similarly to my comment #4286, we aren't actually preventing users from pasting these elements in. If possible, we should restrict users from being able to do this in the first place in CKEditor.